icellmobilsoft/pom.xml

```diff
@@ -109,12 +109,12 @@
 
 			<distributionManagement>
 				<snapshotRepository>
-					<id>ossrh</id>
-					<url>https://oss.sonatype.org/content/repositories/snapshots</url>
+					<id>central</id>
```

Comment:
a profilban mar a "distributionManagement"-re nincs szukseg, a plugin sajat maga kezeli a feltoltest nem a pom-ban definialt szerint. Lasd: https://github.com/nav-gov-hu/parent-pom/blob/main/nav/pom.xml#L110

Reply:
ha nincs megadva a profilban a distributionManagement, akkor azt veszi a central-publishing-maven-plugin, ami a pom-ban van alapból: https://github.com/i-Cell-Mobilsoft-Open-Source/parent-pom/blob/master/icellmobilsoft/pom.xml#L88
tehát a céges nexus-t.

a plugin-t átkonfigurálva a centralSnapshotsUrl-el a snapshot deploy-ra jó, de szerintem release-kor szintén a céges nexus-t fogja venni.

Reply:
bár a ti release-etek kikerült a central-ba :) akkor nem szóltam

Reply:
a publishingServerId befolyásolja? bár az meg default: central: https://central.sonatype.org/publish/publish-portal-maven/#publishingserverid

Reply:
A parent-pom-release profilban mar nem kell distributionManagement mert az uj plugin sajat maga kezeli kodilag. Konfigjai is erre utalnak, sajnis nem teljes a leirasa. En is a forraskodjaban turkaltam hogy megertettem hogy mukodik

A parent-pom/icellmobilsoft/pom.xml onmagaban kell a distributionManagement hiszen akkor az ezt hasznalo projektek nem tudnanak deploy es releaselni ceges repoba. Az egesz parent-pom-release profil csak a parent-pom projektnek kell mert annak ki kell ternie a belso tartalomtol

Reply:
köszi szépen! erre jutottam én is.